.github/workflows/slash-command-armttk.yaml

```diff
@@ -0,0 +1,44 @@
+# THIS WORKFLOW WILL RUN WHEN WE ADD SLASH COMMAND LIKE '/arm-ttk', '/ARM-TTK', '/Arm-Ttk', 'armttk' or 'ARMTTK'
+name: Slash Command ARM-TTK Tests
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  run-arm-ttk:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor != 'dependabot[bot]' && !github.event.pull_request.head.repo.fork && github.event.issue.pull_request && (contains(github.event.comment.body, '/armttk') || contains(github.event.comment.body, '/Armttk') || contains(github.event.comment.body, '/ARM-TTK') || contains(github.event.comment.body, '/ARMTTK') || contains(github.event.comment.body, '/arm-ttk') || contains(github.event.comment.body, '/ArmTtk')) }}
```

Comment:
can't we make this array of string items and check with contains

Reply:
updated code